packages/oui-dual-list/src/dual-list.controller.js

```diff
@@ -0,0 +1,59 @@
+import get from "lodash/get";
+import remove from "lodash/remove";
+
+export default class {
+    constructor ($attrs, $element, $filter, $timeout) {
+        "ngInject";
+
+        this.$attrs = $attrs;
+        this.$element = $element;
+        this.$filter = $filter;
+        this.$timeout = $timeout;
+    }
+
+    $postLink () {
+        this.$timeout(() =>
+            this.$element.addClass("oui-dual-list")
+        );
+    }
+
+    getProperty (item) {
+        return get(item, this.property, item);
```

Comment:
Throw exception ?

Reply:
I think we can keep how it works for now, even if it will display the object with the json filter.
It's more helpful than throwing an error for a bad property path :)